README.md

```diff
@@ -1,3 +1,16 @@
+<Remove before land>
+Instructions to test `iFirstPersonControls` locally: 
```

Comment:
We should totally add a markdown for an introduction on how to contribute, thanks for basically bringing that up here.